diagrams/src/diagrams_builder.js

```diff
@@ -1,17 +1,17 @@
 (function(root, factory) {
     if (typeof define === 'function' && define.amd) {
         // AMD. Register as an anonymous module.
-        define(['../vendor/railroad-diagrams'], factory)
+        define(['../vendor/railroad-diagrams', '../../bin/chevrotain'], factory)
```

Comment:
The relative path here won't work for the bower release. The problem is that bower uses github.com as it's repository in addition to chevrotain.js being a generated file. I will have to reorganize the file structure so it would be identical in both bower and npm first.